static/usage.html

```diff
@@ -460,11 +460,11 @@ <h3><a href="#updates-security">Security</a></h3>
 
                     <p>The update server isn't a trusted party since updates are signed and verified along
                     with downgrade attacks being prevented. The update protocol doesn't send identifiable
-                    information to the update server and works well over a VPN / Tor. GrapheneOS isn't
+                    information to the update server and works well over a VPN/Tor. GrapheneOS isn't
                     able to comply with a government order to build, sign and ship a malicious update to a
                     specific user's device based on information like the IMEI, serial number, etc. The
                     update server only ends up knowing the IP address used to connect to it and the
-                    version being upgraded from based on the requested incremental.</p>
+                    version being upgraded based on the requested incremental.</p>
```

Comment:
I personally think keeping the "from" makes it clearer.

Reply:
I guess you mean replacing "from" from "based" right ?

Reply:
It seems redundant to me to have both words, I thought it was a mistake actually.

Reply:
I mean I don't think this part needs to be changed.

Reply:
I really find it weird, so weird that I don't really understand the end of the sentence with "from" in it. That could be me tbf. 

If you think so, let's keep it then.

Reply:
@d4rklynk In short, I think this section should remain unchanged for now.

It's very important information and it needs to be 100% accurate as well as very clear. I think the feedback that you've given is helpful. It's very likely that others who have read this section have also found the wording to be unclear, which means it should be addressed.

Due to its importance, I think any change to this section should be its own pull request and the wording should be reviewed by multiple people.

Reply:
I understand, thanks for explaining to me.